### DISCUSSION TITLE
Use client local /home instead of SSHFS or NFS mount ?

### DISCUSSION THREAD

**Discussion by SimStone** [ltsp/ltsp#549] [Q&A] [answered]
Hello,

In my company we currently provide users access to xubuntu 16.04 fat-clients using LTSP5 and PXE boot, the image being served with NBD.
We want to upgrade to the last xubuntu LTS version and have begin testing the new LTSP. 
We would like to know if it's possible to use the client local /home instead of using SSHFS or NFS mounting (with LTSP5 and NBD when users download or delete files those modifications stay local on the client, there is no impact on the server and that's something we want to keep).

**Comment by alkisg** [comment, marked as answer]
Yup, it's just a line in ltsp.conf, for example:

```shell
[clients]
FSTAB_HOME="LABEL=home	/home		ext4	defaults	0	2"
```

This will mount any partition that you have labelled "home" (so that you don't have to care about UUIDs or partition numbers) into /home, and then LTSP will automatically use it.

**Reply by SimStone** [reply]
ok I'll do that. Thanks

**Comment by MacMilanzi** [comment]
Hi, I have the similar question here, I added the line you mentioned in the ltsp.conf but the thin client gets stuck at the picture below for a while then Times out. When it gets to the login screen and i login the screen is blank, and when i reboot it gets stuck on the second picture below. Can you help? I would like to run from the local /home of the thin client after boot without using sshfs or nfs. This partition you are mentioning is it on the ltsp server? Can the client run independently without the server connection after boot? like changes on the thin client doesnt affect the server or write to the server.
![IMG_8248](https://github.com/user-attachments/assets/9b341854-1491-49af-82c6-551275ae24ee)
![IMG_8249](https://github.com/user-attachments/assets/3a95dbbd-1f2f-4a5d-8f43-14440653534f)
